utils/ansible/ftest/templates/daos-launch.sh.j2

```diff
@@ -248,12 +248,16 @@ common_opts="-ar --provider={{ daos_ofi_provider | default(daos_ofi_provider_def
 {% endif %}
 
 info "Starting functional test simulation"
+source "$DAOS_INSTALL_DIR/bin/activate"
 {% if daos_ofi_interface is defined %}
-env OFI_INTERFACE=$OFI_INTERFACE $PYTHON_EXE launch.py $common_opts "${args[@]}"
+run env OFI_INTERFACE=$OFI_INTERFACE $PYTHON_EXE launch.py $common_opts "${args[@]}"
 {% else %}
-$PYTHON_EXE launch.py $common_opts "${args[@]}"
+run $PYTHON_EXE launch.py $common_opts "${args[@]}"
 {% endif %}
 rc=$?
+set +u
+deactivate
+set -u
```

Comment:
FYI - if you copied this from my code then when using a later python version than 3.6 it's the +u/-u settings are no longer required.  @phender this will apply to your PR as well, if using a newer python then you could remove these extra lines.

Reply:
I read the comment about 3.6 or higher, I will remove it as it is clearly "belt and braces" as we say in France ;-)

- [ ] Remove useless set +u before activate

Reply:
In fact I can elaborate further.  Phil is upgrading the avocado version in https://github.com/daos-stack/daos/pull/14148 and that requires a newer python which them means that the python bug doesn't work with `+u` set will no longer apply.  For this PR you should IMO also use a newer python version (there's no reason not to use the latest version available in the distro which is probably 3.11)  even though it may not be strictly needed today - otherwise you'll have to rework this soon.

The `-u` changes needed applies to all instances of the `deactivate` command in this PR obviously.

Reply:
> I will remove it as it is clearly "belt and braces" as we say in France ;-)
> 
> * [x]  Remove useless set +u before activate

Fixed with commit ad2c133d7

Reply:
Awesome.